distributedlog-build-tools/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
```

Comment:
I moved the eclipse setting to the build-tools module. I will create a jira for findbugs settings.